public/base-locales/de.json

```diff
@@ -109,19 +109,19 @@
   "allPokestops": "Alle Pokestops",
   "lures": " Lockmodule",
   "quests": "Quests",
-  "invasions": "invasion",
-  "portals": "Portals",
+  "invasions": "Invasionen",
+  "portals": "Portale",
   "submissionCells": "Zellen übermitteln",
   "spawnpoint": "Spawnpunkt",
   "spawnpoints": "Spawnpunkte",
   "s2CellLevel": "Level {{level}} S2 Zellen",
   "totalCount": "Komplette Anzahl",
   "submissions": "{{}} ",
```

Comment:
this one is missing in english!?

Reply:
No?

https://github.com/WatWowMap/ReactMap/blob/191081fe4aaaa4d8a3a544a9ab0f84a858df5daf/public/base-locales/en.json#L114